qiita_ware/analysis_pipeline.py

```diff
@@ -28,6 +28,9 @@ def _build_analysis_files(analysis, r_depth=None, **kwargs):
     r_depth : int, optional
         Rarefaction depth for biom table creation. Default None
     """
+    if not analysis.jobs:
+        raise RuntimeError("Analysis %d has no jobs attached!" % analysis.id)
```

Comment:
Should never be shown to them. This is more for CLI interface we were talking about where you might try and run an analysis with no jobs.